src/R/Components/Impl/ConnectionManager/IConnection.cs

```diff
@@ -8,8 +8,9 @@ namespace Microsoft.R.Components.ConnectionManager {
     public interface IConnection {
         Uri Id { get; }
         string Name { get; }
+        string Path { get; }
```

Comment:
What's the distinction between `Id`, `Name` and `Path`? For remote connections in particular.